tests/integration/v3_watch_restore_test.go

```diff
@@ -97,12 +119,22 @@ func TestV3WatchRestoreSnapshotUnsync(t *testing.T) {
 		}
 	}
 
+	// ensure two members has compacted the log twice.
+	if err := logSub.wait(5 * time.Second); err != nil {
+		t.Fatal("Failed to ensure that members compacted Raft log in 5 seconds")
+	}
+	logMonitor.delSubscriber(logTopic)
+	t.Logf("two members have compacted raft logs")
+
 	// trigger snapshot send from leader to this slow follower
```

Comment:
nit: replace with `After RecoverPartition, leader L will send snapshot to slow F_m0 follower, because  F_m0(index:8) is 'out of date' compared to L(compacted-index:17)`

Reply:
Nice comment. Updated!